technical_overview/technical_details/Message_flow.md

```diff
@@ -0,0 +1,60 @@
+# Message flow through EMAP pipeline
+As detailed in the [technical overview](../Technical_overview_of_EMAP.md), the purpose of the Experimental Medicine
+Application Platform (EMAP) is to offer real-time clinical data suitable to analysis, such as the prediction of ICU 
+bed occupancy based on current workloads and planned surgeries. This requires the pipeline to consolidate historical
+data with live information available through message streams. 
+
+Messages on these message streams hold varying kinds of data, including patient details such as name and medication 
```

Comment:
We don't actually have a medications stream at the moment, might be better to say test results or observations?
```suggestion
Messages on these message streams contain diverse data, including patient details such as name and medication 
```